terraform/main.tf

```diff
@@ -0,0 +1,5 @@
+provider "aws" {
```

Comment:
Are the contents of this file actually doing anything at the moment? Is the main.tf file necessary?

Reply:
The provider has to be defined somewhere, if not here then in on of the random resources I picked and I didn't want that place to be arbitrary such as, defining the provider in the db.tf. So main.tf made sense.